src/Markdown.MAML/Parser/MarkdownParser.cs

```diff
@@ -148,7 +148,15 @@ private void InitializePatternList()
 
                         {   "italic",
                             @"\*(.+?)\*",
-                            this.CreateItalicSpan }
+                            this.CreateItalicSpan },
+
+                        {   "bold2",
+                            @"(?<![a-zA-Z0-9])__(.+?)__(?![a-zA-Z0-9])",
```

Comment:
@vors there is a problem where some words will have the underscores inside, and the pattern was matching them, when it shouldn't have.
IE: `about_some_thing`
`_some_` was a match when it should not have been.